facebook/Docusaurus
# feat: code block tab

## Motivation

#103 

### Have you read the [Contributing Guidelines on pull requests](https://github.com/facebook/Docusaurus/blob/master/CONTRIBUTING.md#pull-requests)?

Yes

## Test Plan

1. I currently changed custom-pages in the 1.0.15 version so that I can visualize how it looks like, I plan to remove it later. So, go to /docs/en/1.0.15/custom-pages:
<img width="1792" alt="screen shot 2018-10-24 at 11 12 02 pm" src="https://user-images.githubusercontent.com/24544912/47441243-4a10dd00-d7e2-11e8-850b-cd917c2b9741.png">
2. Try to click on the tab
3. See if the other code tab changes too or not
4. Try to resize the width of the browser to be abit smaller to see how it looks like if it overflow

## Related PRs

Null

## Todo 
1. Is the design suitable? Should we handle older browser that might not support horizontal scroll?
2. Should we have other design for smaller device?
3. Is the way I add the onclick in the js below the copy field looks right?
